base/Dockerfile

```diff
@@ -66,3 +66,5 @@ RUN ./install_area_detector.sh
 
 COPY install_motor.sh .
 RUN ./install_motor.sh
+
+COPY lnls-prune-artifacts.sh /usr/local/bin/lnls-prune-artifacts
```

Comment:
I've added this last so that image cache is not invalided everything for nothing. It can be moved to be side-by-side with `lnls-run` after the script ready to be merged.

Reply:
Made this a permanent change now. ;)